src/masternodes/masternodes.h

```diff
@@ -287,6 +287,7 @@ struct CCollateralLoans {
     std::vector<CTokenAmount> loans; // in USD
 
     uint32_t ratio() const {
+        if (loans.empty()) return 0;
```

Comment:
Return `std::numeric_limits<uint32_t>::max` because when only collaterals are in ratio is the higher one.